.github/actions/find-comment/action.yml

```diff
@@ -0,0 +1,15 @@
+name: 'Find comment'
+description: 'Finds the id of the first comment by the GitHub Actions bot.'
+runs:
+  using: 'node12'
+  main: 'main.js'
+inputs:
+  author:
+    description: |
+      The name of the author to search for in the list of comments to update.
+      This task will find the id of the first comment by this author.
```

Comment:
... ", or the blank string if it's not made a post yet."

(to distinguish from an error case)